glide.yaml

```diff
@@ -9,3 +9,7 @@ import:
 - package: github.com/spf13/cobra
 - package: github.com/xeipuuv/gojsonschema
   version: d5336c75940ef31c9ceeb0ae64cf92944bccb4ee
+- package: github.com/russross/blackfriday
```

Comment:
not sure how that will work with the existing `update-deps` makefile target - because I run the above, then `make update-deps` and automagically `glide-vc` removes those test vendor files (the ones under vendor/)